### PR TITLE
feat(/xref): expose `htmlProse` field from webref

### PR DESCRIPTION
**Pull request by dontcallmedom**
See https://github.com/w3c/respec/issues/4522#issuecomment-1933574282 for context

### CODE REVIEW

File: tests/routes/xref/lib/search.test.js [hunk 1]
```diff
@@ -60,6 +60,9 @@ describe("xref - search", () => {
             spec: "font-metrics-api-1",
             type: "interface",
             uri: "#baseline",
+	    normative: true,
+	    htmlProse: "test html Prose",
+	    for: undefined,
```
Comment (sidvishnoi) [inline tests/routes/xref/lib/search.test.js:65]: Nit: prettier?

**Review by sidvishnoi — APPROVED** [review]
My only concern here is with the size of `xref.json` file, which is already 11MB in size. It gets loaded on https://respec.org/xref/filter and as well as reloaded in webserver on cache-refresh.

I think we can leave it as an afterthought, as xref backend needs some work anyway (re: Edge server and all). But @dontcallmedom, can you tell the updated size of generated xref.json file?

**Comment by sidvishnoi** [comment]
Merging and release for now though.

**Comment by dontcallmedom** [comment]
renegerating it locally, I get 11949774 bytes for xref.json, so not much of a difference yet, but the risk that it grows significantly as other specs adopt the convention is real

**Pull request merged**